aiohttp test failures

We have [started to see the following failures](https://apm-ci.elastic.co/job/apm-agent-python/job/apm-agent-python-nightly-mbp/job/master/775/) in the nightly tests:

    Test / Python-python-3.9-0 / tests.contrib.asyncio.aiohttp_web_tests.test_http_exception_below_500[pyloop]
    Test / Python-python-3.8-3 / tests.contrib.asyncio.aiohttp_web_tests.test_http_exception_below_500[pyloop]
    Test / Python-python-3.7-3 / tests.contrib.asyncio.aiohttp_web_tests.test_http_exception_below_500[pyloop]

An example stacktrace from a failed test looks like this:

```
aiohttp_client = <function aiohttp_client.<locals>.go at 0x7f7b81ad5280>
aioeapm = <elasticapm.contrib.aiohttp.ElasticAPM object at 0x7f7b81d79760>

    async def test_http_exception_below_500(aiohttp_client, aioeapm):
        app = aioeapm.app
        client = await aiohttp_client(app)
        elasticapm_client = aioeapm.client
        resp = await client.get("/raise/ok")
>       assert resp.status == 200
E       assert 500 == 200
E         +500
E         -200

tests/contrib/asyncio/aiohttp_web_tests.py:156: AssertionError
```

Possible culprit? https://github.com/elastic/apm-agent-python/pull/1384

--- a/.ci/.jenkins_framework_full.yml
+++ b/.ci/.jenkins_framework_full.yml
@@ -65,7 +65,6 @@ FRAMEWORK:
   - gevent-newest
   - zerorpc-0.4
   - aiohttp-3.0
-  - aiohttp-4.0
   - aiohttp-newest
   - aiopg-newest
   - asyncpg-newest

--- a/tests/requirements/reqs-aiohttp-4.0.txt
+++ b/tests/requirements/reqs-aiohttp-4.0.txt
@@ -1,2 +1,0 @@
-aiohttp>=4.0.0a1,<5.0
--r reqs-base.txt

